docs/sources/reference/components/beyla/beyla.ebpf.md

```diff
@@ -128,12 +169,15 @@ In some scenarios, Beyla instruments a wide variety of services, such as a Kuber
 The `services` block allows you to filter the services to instrument based on their metadata. If you specify other selectors in the same services entry,
 the instrumented processes need to match all the selector properties.
 
-| Name         | Type     | Description                                                                     | Default | Required |
-| ------------ | -------- | ------------------------------------------------------------------------------- | ------- | -------- |
-| `exe_path`   | `string` | The path of the running service for Beyla automatically instrumented with eBPF. | `""`    | no       |
-| `name`       | `string` | The name of the service to match.                                               | `""`    | no       |
-| `namespace`  | `string` | The namespace of the service to match.                                          | `""`    | no       |
-| `open_ports` | `string` | The port of the running service for Beyla automatically instrumented with eBPF. | `""`    | no       |
+| Name                                  | Type    | Description                                                     | Default | Required |
+| ------------------------------------- | ------- | --------------------------------------------------------------- | ------- | -------- |
+| `exe_path`                           | `string` | The path of the running service for Beyla automatically instrumented with eBPF. | `""`    | no       |
+| `name`                               | `string` | The name of the service to match.                               | `""`    | no       |
+| `namespace`                          | `string` | The namespace of the service to match.                          | `""`    | no       |
+| `open_ports`                         | `string` | The port of the running service for Beyla automatically instrumented with eBPF. | `""`    | no       |
+| `skip_go_specific_tracers`            | `bool`  | Skip Go-specific tracers during discovery.                      | `false` | no       |
+| `exclude_otel_instrumented_services`  | `bool`  | Exclude services that are already instrumented with OpenTelemetry.| `true`  | no       |
+| `default_exclude_services`            | `list(string)` | List of services to exclude by default.                 | `["(?:^\|\/)(beyla$\|alloy$\|otelcol[^\/]*$)"]`    | no       |
```

Comment:
Where is this set? I couldn't see it. 

Nit: It looks like it supports regex. 

Nit 2: maybe it's better to have 3 entries for beyla, alloy, otelcol instead of one entry for all 3.

Reply:
> Where is this set? I couldn't see it.

line 126 of  `beyla_linux.go` 

> Nit: It looks like it supports regex. Should we document this?

Sure, will do!
 
> Nit 2: maybe it's better to have 3 entries for beyla, alloy, otelcol instead of one entry for all 3.

That is the default in Beyla right now. I'd rather not change it if that's fine for you.

Reply:
actually this section doesn't belong here, good that you pointed out!